flows/photometry.py

```diff
@@ -617,8 +644,10 @@ def photometry(fileid, output_folder=None, attempt_imagematch=True):
 	tab.meta['pixel_scale'] = pixel_scale * u.arcsec/u.pixel
 	tab.meta['seeing'] = (fwhm*pixel_scale) * u.arcsec
 	tab.meta['obstime-bmjd'] = float(image.obstime.mjd)
-	tab.meta['zp'] = zp
-	tab.meta['zp_error'] = zp_error
+	tab.meta['zp'] = zp_bs
```

Comment:
May I suggest to not add BOTH of them to the headers... I think it will just be a source of confusion... "Which one was actually used?"

Reply:
It's actually only tracking the difference between the two ZPs, so it shouldn't be confusing. The reason to add them both, for now, is to faciliate comparison. But I'm only tracking the quantity of interest, which is the zp difference. 

The bootstrapped zp is always being used, but I think it would be good to track the "naive" zp as well, at least for monitoring sake.  We want to see if the bootstrapped version is significantly different, and in what scenarios, as it's a bit of a novel/untested way of determining the zp and zp error. I want to study any scenarios where they differ significantly. We may even end up scrapping the bootstrapped version if we find problematic edge cases that we can't resolve.

Reply:
If you agree, I suggest leaving it as is.